### PR TITLE
Drop support for Python 3.5

### PR DESCRIPTION
**Pull request by francoisfreitag**
This version has reached its end of life.

### CODE REVIEW

File: factory/base.py [hunk 1]
```diff
@@ -696,10 +696,7 @@ def _build(cls, model_class, *args, **kwargs):
             raise ValueError(
                 "ListFactory %r does not support Meta.inline_args." % cls)
 
-        # When support for Python <3.6 is dropped sorting will no longer be required
-        # because dictionaries will already be ordered, this can then be changed to:
-        # values = kwargs.values()
-        values = [v for k, v in sorted(kwargs.items(), key=lambda item: int(item[0]))]
+        values = kwargs.values()
```
Comment (rbarrois) [inline factory/base.py:699]: Maybe we should keep a comment reminding ourselves that this is sorted anyway through builtins?

File: tox.ini [hunk 2]
```diff
@@ -2,8 +2,8 @@
 minversion = 1.9
 envlist =
     lint
-    py{35,36,37,38}-django111-alchemy-mongoengine,
-    py{35,36,37,38}-django22-alchemy-mongoengine,
+    py{36,37,38}-django111-alchemy-mongoengine,
```
Comment (rbarrois) [inline tox.ini:5]: I think we should also drop Django 1.11 :)

**Review by rbarrois — APPROVED** [review]

File: factory/base.py (hunk 1 above)
Comment (francoisfreitag) [reply factory/base.py:701]: I added a comment, please let me know what you think.

File: tox.ini (hunk 2 above)
Comment (francoisfreitag) [reply tox.ini:5]: Yep, done in #793.

**Comment by rbarrois** [comment]
Merged as 64f10a18525572a5158b753d8f20195553fa5c5f !

**Pull request closed without merging**